Update Chinese l18n (1.3.4 dev2)

hedw1gP opened the pull request.

hedw1gP committed:
Update Chinese l18n (1.3.4 dev2)

hedw1gP commented on README_CN.md:
```diff
@@ -24,9 +24,11 @@
-<a href="https://youtu.be/_vfZYtiRfRs?list=PL6UPd2Tj65nFLGMBqKaeKOPNp2HOO86Uw"><img src="https://img.youtube.com/vi/_vfZYtiRfRs/0.jpg"></a> 
+<a href="https://youtu.be/5Tnp4z6GWME?list=PL6UPd2Tj65nFLGMBqKaeKOPNp2HOO86Uw"><img src="https://img.youtube.com/vi/5Tnp4z6GWME/0.jpg"></a> 
+
+也可前往 [McHorse's Mods 哔哩哔哩中文频道](https://space.bilibili.com/472615413) 观看。
```
Add a link to Bilibili channel

mchorse commented:
Looks good, however, can you also edit [this](https://github.com/mchorse/aperture/compare/1.3.3...master#diff-6ebf47e5765faf9f5ecf471eef6e26b0R30)?

Chunk7w committed:
Update error.yml

Chunk7w committed:
Update gui.yml

hedw1gP commented:
Thank you so much! @Chunk7w

Chunk7w committed:
Updated Chinese translation

把 not_exists: "{f}在当目前相机配置文件里没有相机关键点在 %s{f} 索引号里！"
改为了 not_exists: "{f}在当前的相机配置文件里没有相机关键点在 %s{f} 索引号里！"

Chunk7w committed:
Fix "目前" to "当前"

Chunk7w committed:
Update zh_TW translation

Chunk7w committed:
Update zh_TW translation

Chunk7w committed:
Update zh_TW translation

mchorse commented:
Awesome, guys, thank you very much! 👏 😁
